Frends.Template/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [1.0.0] - 2023-08-08
```

Comment:
If we are overriding the old package we should have some other version number and some change notes

Reply:
We are not